compiler/testData/diagnostics/tests/callableReference/kt32267.fir.kt

```diff
@@ -0,0 +1,39 @@
+// SKIP_TXT
+// !DIAGNOSTICS: -UNUSED_PARAMETER
+// !LANGUAGE: +NewInference
+
+fun main() {
+    Configuration().commands {
+        Command1 { <!UNRESOLVED_REFERENCE!>someService::execute<!> } // Overload resolution ambiguity. All these functions match.
+        Command2 { <!UNRESOLVED_REFERENCE!>someService::execute<!> } // Overload resolution ambiguity. All these functions match.
```

Comment:
So, these two break `// FIR_IDENTICAL`, and indeed, these are `FirNamedReference` without pointing any target/candidate symbol. This will fail eventually if you compile this.